Frends.MySQL.ExecuteQuery/Frends.MySQL.ExecuteQuery.Tests/UnitTests.cs

```diff
@@ -0,0 +1,151 @@
+using Frends.MySQL.ExecuteQuery.Definitions;
+using Microsoft.VisualStudio.TestPlatform.CoreUtilities.Extensions;
+using MySql.Data.MySqlClient;
+using NUnit.Framework;
+namespace Frends.MySQL.ExecuteQuery.Tests;
+
+/// <summary>
+/// Setup MySQL to docker:
+/// docker run --name some-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=my-secret-pw -d mysql

```

Comment:
Please remove `--name some-mysql` so the container name will be automatically generated.